Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```diff
@@ -399,6 +399,9 @@
     - state: ai_camera
       shader: unshaded
       map: ["base"]
+  - type: Tag
+    tags:
+    - StationAi
```

Comment:
The StationAi tag is already used for the AiHolder mind slot as a whitelist, so you should not reuse that here or it might cause bugs.
Best make a new tag and use that both for the AI and the AdminObserver.

Reply:
both should be done